src/legacy/core_plugins/data/public/index.ts

```diff
@@ -81,4 +81,7 @@ export {
   // search_source
   getRequestInspectorStats,
   getResponseInspectorStats,
+  // tabify
```

Comment:
Also, if we're at it. what **is** tabify?
As this is pretty old code, would you mind adding a couple of sentences on what it is used for?